examples/api/lib/widgets/custom_multi_child_layout/custom_multi_child_layout.0.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file should be named (and moved to) `examples/api/lib/widgets/basic/custom_multi_child_layout.1.dart` since it will be linked into the `basic.dart` file.

A link to it should also be added below this line in the API doc comments:
https://github.com/flutter/flutter/blob/dffddf00a22e77122d67f4503efef16a46c18ec2/packages/flutter/lib/src/widgets/basic.dart#L2213